src/NuGet.Core/NuGet.Build.Tasks.Pack/DefaultSymbolsPackageFormatChangingWarningTask.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using Microsoft.Build.Utilities;
+using NuGet.Common;
+
+namespace NuGet.Build.Tasks.Pack
+{
+    public class DefaultSymbolsPackageFormatChangingWarningTask : Task
+    {
+        public ILogger Logger => new MSBuildLogger(Log);
+
+        public override bool Execute()
+        {
+            Logger.Log(LogMessage.CreateWarning(NuGetLogCode.NU5132,
```

Comment:
Would this get elevated to an error? 

I'd imagine no is the answer here. 
We should probably raise the warning from the PackTask where warnings as errors and no warn are respected.